### PR TITLE
feature/ziggeo video recorder

### PR DESCRIPTION
**Pull request by mrJmek**
Перейменував гілку, тому старий pull request тут: https://github.com/superNazo/video_quiz/pull/6

Документація: https://ssl.ziggeo.com/docs
Офіційний GitHub репозиторій: https://github.com/Ziggeo/ZiggeoRubySdk

- [ ] @vovka 
- [x] @superNazo 
- [x] @AntonMayer

### CODE REVIEW

File: app/controllers/video_recorder_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class VideoRecorderController < ApplicationController
+  before_action :ziggeo_init, :all_videos, only: :index
+
+  def index
+  end
+
+  private
+
+  def ziggeo_init
+    @ziggeo = Ziggeo.new('1eb1e578bfc596f031a607decb12685c',
```
Comment (vovka) [inline app/controllers/video_recorder_controller.rb]: Can we move this sensitive information to some config files and not commit it?

**Comment by vovka** [comment]
Removed all "Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping" hound comments. Other have to be respected.

File: app/controllers/video_recorder_controller.rb (hunk 1 above)
Comment (mrJmek) [inline app/controllers/video_recorder_controller.rb]: Done!
Переніс токени у файл: config/secrets.yml

File: app/helpers/video_recorder_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module VideoRecorderHelper
```
Comment (vovka) [inline app/helpers/video_recorder_helper.rb]: Could you please also remove this empty file? Let's not commit blank files.

**Comment by vovka** [comment]
And one more little thing. There are plenty of commits. Could you please join them all in one. It could be done with `git rebase -i` command using `squash` action. See https://git-scm.com/book/ru/v1/%D0%98%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D1%8B-Git-%D0%9F%D0%B5%D1%80%D0%B5%D0%B7%D0%B0%D0%BF%D0%B8%D1%81%D1%8C-%D0%B8%D1%81%D1%82%D0%BE%D1%80%D0%B8%D0%B8#Уплотнение-коммитов

File: app/helpers/video_recorder_helper.rb (hunk 2 above)
Comment (mrJmek) [inline app/helpers/video_recorder_helper.rb]: Opps.
I previously used this helper, but now it's useless...

**Pull request merged**